test/integration/extensions_test.go

```diff
@@ -0,0 +1,229 @@
+package integration
+
+import (
+	"bytes"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	testutil "github.com/openshift/origin/test/util"
+)
+
+func TestExtensions(t *testing.T) {
+	// Create a temporary directory.
+	tmpdir, err := ioutil.TempDir("", "extensions")
+	if err != nil {
+		t.Fatalf("Could not create tmp dir for extensions: %v", err)
+	}
+	defer os.RemoveAll(tmpdir)
+
+	// Create extension files.
+	masterOptions := setupExtensions(t, tmpdir)
+
+	// Start server.
+	_, err = testutil.StartConfiguredMaster(masterOptions)
+	if err != nil {
+		t.Fatalf("Unexpected error starting server: %v", err)
+	}
+
+	// Request the extension scripts.
+	expectedScripts := concat(";\n", script1, script2)
```

Comment:
don't duplicate concat in the test, just expect `script1 + separator + script2`... you could even export the script and css separators from the assets package as constants you could reference here